src/main/java/strategy/handler/ReturnOperationHandler.java

```diff
@@ -0,0 +1,19 @@
+package strategy.handler;
+
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
```

Comment:
Do the same for ReturnOperationHandlerImpl and SupplyOperationHandlerImpl as for BalanceOperationHandlerImpl and PurchaseOperationHandlerImpl

Reply:
fixed